mycore-iiif/src/main/java/org/mycore/iiif/image/MCRIIIFImageUtil.java

```diff
@@ -28,14 +26,26 @@
 import org.mycore.iiif.image.impl.MCRIIIFImageImpl;
 import org.mycore.iiif.image.model.MCRIIIFImageProfile;
 
+import static org.mycore.iiif.image.resources.MCRIIIFImageResource.IIIF_IMAGE_API_2_LEVEL2;
+
 public class MCRIIIFImageUtil {
     public static void completeProfile(MCRIIIFImageImpl impl, MCRIIIFImageProfile profile) {
         profile.setId(getProfileLink(impl));
     }
 
+    public static String encodeURIComponent(String component) {
```

Comment:
Please provide some Javadoc, WHY we handle some special characters different here.
Why not rename "\\+" to " " (space)  instead of "%20"?
Can we add "%2F" -> "/" - I have slashes in my IDs (PURLs) - or would this break image path calculations.